tools/plugins/com.liferay.ide.upgrade.core/src/com/liferay/blade/upgrade/liferay70/BREAKING_CHANGES.markdown

```diff
@@ -4341,4 +4341,36 @@ You should change the package name via "correct automatically" in liferay-ide up
 
 #### Why was this change made? [](id=why-was-this-change-made-104)
 
-To leverage the benefits of modularization in Liferay 7.
\ No newline at end of file
+To leverage the benefits of modularization in Liferay 7.
+
+---------------------------------------
+
+### Changed the BaseAssetRenderer APIs getIconPath Method ThemeDisplay Parameters [](id=changed-the-baseassetrenderer-apis-geticonpath-method-themedisplay-parameters)
```

Comment:
hey @codyhoag  while testing code upgrade tool with demo project we found this was a breaking change that was missed.  Do we have this documented somewhere?

Reply:
@gamerson not that I know of. Is this something you could add to the [Breaking Changes](https://github.com/liferay/liferay-portal/blob/master/readme/7.0/BREAKING_CHANGES.markdown) doc? This is for 7.0 and not 7.1, right?